tools/serverpod_cli/lib/src/migrations/generator.dart

```diff
@@ -364,24 +396,28 @@ class MigrationGenerator {
 
 class MigrationVersion {
   MigrationVersion({
-    required this.migrationsDirectory,
+    required this.moduleName,
+    required Directory versionDirectory,
     required this.versionName,
     required this.migration,
     required this.databaseDefinition,
-  });
+  }) : _versionDirectory = versionDirectory;
 
-  final Directory migrationsDirectory;
+  final Directory _versionDirectory;
+
+  final String moduleName;
   final String versionName;
   final DatabaseMigration migration;
   final DatabaseDefinition databaseDefinition;
   int get priority => migration.priority;
 
   static Future<MigrationVersion> load({
+    required String moduleName,
     required String versionName,
-    required Directory migrationsDirectory,
+    required Directory baseDir,
```

Comment:
Suggestion(NIT): 
```suggestion
    required Directory migrationDirectory,
```

Since this is the directory that holds all the migrations.